src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
```

Comment:
Was this necessary?  If yes, can you move it down a line?  But I'd also have expected it to not be needed, since everything below is in the System.Net.Sockets namespace, and thus System should be implicitly included.